Dockerfile.rhel

```diff
@@ -0,0 +1,20 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-etcd-golang-1.12 AS builder
```

Comment:
Should we switch to 1.15 or do you want to do that in separate PR?

Reply:
yeah lets merge this as 1.12 them pivot to go1.15

Reply:
well on second thought I will change this then pivot in release.